jaeger/charts/linkerd-jaeger/templates/namespace-metadata-rbac.yaml

```diff
@@ -50,3 +50,26 @@ subjects:
 - kind: ServiceAccount
   name: namespace-metadata
   namespace: {{.Release.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Values.linkerdNamespace }}
+  {{- with .Values.commonLabels }}
+  labels: {{ toYaml . | trim | nindent 4 }}
+  {{- end }}
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+    {{- /* cross-namespace RBAC doesn't play well as a hook for some reason */}}
```

Comment:
can you clarify this?  why do we need the helm hook annnotations on the RoleBinding? why are they commented out?  what is the consequence of cross namespace RBAC not working for hooks?

Reply:
Thanks for pointing this out. The problem was apparently related to ordering, i.e. the script in the namespace job running before it acquired the permissions to access the ConfigMap. I uncommented these lines, and set `helm.sh/hook-weight` to 1 in the namespace jobs so they run after these RoleBindings are persisted.